### PR TITLE
Devi/memory status

### PR DESCRIPTION
**Pull request by v-dmudili**
Created a folder for Memory Status app & added all the project files for windows 10 Iot Enterprise.

Following changes are done : 

1) file modified : MemoryStatus.vcxproj
    Updated the Platform Toolset  from  'v141 to 'v142'.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Generate and installing app package procedure.
    Updated the Windows Platform from IoT Core to IoT Enterprise

### CODE REVIEW

File: MemoryStatus/cpp/MemoryStatus.vcxproj [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{95BE87FD-89C1-4238-95D3-FA0F55FFBE8B}</ProjectGuid>
+    <SccProjectName>SAK</SccProjectName>
+    <SccAuxPath>SAK</SccAuxPath>
+    <SccLocalPath>SAK</SccLocalPath>
+    <SccProvider>SAK</SccProvider>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>MemoryStatus</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```
Comment (neilsh-msft) [inline MemoryStatus/cpp/MemoryStatus.vcxproj]: This was changed from 10.0.17763.0 in the original repo. Is this intended?

Comment (v-dmudili) [reply MemoryStatus/cpp/MemoryStatus.vcxproj]: It has changed Automatically, it was not intended.

Comment (rsameser) [reply MemoryStatus/cpp/MemoryStatus.vcxproj]: @v-dmudili please address

**Review by rsameser — APPROVED** [review]

File: MemoryStatus/cpp/MemoryStatus.vcxproj (hunk 1 above)
Comment (v-abdrahman) [reply MemoryStatus/cpp/MemoryStatus.vcxproj]: Updated the Platform Version to 10.0.19041.0

**Pull request merged**